contracts/oracle_implementation/library.cairo

```diff
@@ -52,7 +52,8 @@ func Oracle_only_oracle_controller{
         # Assume uninitialized
         return ()
     end
-    with_attr error_message("OracleImplementation: This function can only be called by the oracle controller"):
+    with_attr error_message(
+            "OracleImplementation: This function can only be called by the oracle controller"):
```

Comment:
Make sure you have formatting set up locally, I think this was changed from your PR a while back @cheikhfiteni

Reply:
And as a follow up we have this ticket: https://www.notion.so/42labs-xyz/Set-up-GHA-for-linting-cairo-code-7631ff8b162548a0ac2df01922977f62